zokrates_cli/src/helpers.rs

```diff
@@ -1,6 +1,5 @@
-use core::convert::TryFrom;
-
 use crate::constants::*;
+use core::convert::TryFrom;
```

Comment:
I don't think we use the core crate anywhere else, would `std::convert::TryFrom` work here?

Reply:
Good catch, I guess this was an auto-import by the IDE, will change to `std::convert::TryFrom`